src/components/FavoriteServices/LandingNavFavorites.scss

```diff
@@ -0,0 +1,25 @@
+@import "~@redhat-cloud-services/frontend-components-utilities/styles/_all";
+@import '~@patternfly/patternfly/patternfly-addons.scss';
+
+.pf-c-tile {
+  margin-left: 50px !important;
+  margin-right: 50px !important;
+  width: 204px;
+  height: 90px;
+}
+.hr-c-tile-landing {
+  --pf-l-gallery--m-gutter--GridGap: 10px !important;
+}
+
+.chr-c-icon-favorites {
+  padding-right: 18px !important;
+}
+
+.pf-c-options-menu__toggle {
+  display: none !important;
```

Comment:
This will remove every toggle, which is not good. Any reason for it?

Reply:
This is to remove the dropdown for the pagination since its not needed. I saw no other way to do this. Would this be ok if I put the pagination in its own class?

Reply:
@karelhala does the class I added in pagination fix that problem?